nncf/quantization/fake_quantize.py

```diff
@@ -21,6 +21,9 @@
 from nncf.common.quantization.structs import QuantizerConfig
 from nncf.common.quantization.structs import QuantizerGroup
 from nncf.common.tensor_statistics.statistics import MinMaxTensorStatistic
+from nncf.experimental.tensor import Tensor
+from nncf.experimental.tensor import TensorDataType
+from nncf.experimental.tensor import functions as fns
```

Comment:
Looks like we may try to implement all these entities in the one namespace as done in the `numpy`.
Because for now, we should always keep in mind that we can't do anything with the `Tensor` without importing `fns` everywhere, which is not convenient as may be.

Reply:
Yes, i have suggest to move `nncf` scope (ex. `nncf.Tensor`, `nncf.DataType`, `nncf.mean`), but not sure that we should do it while Tensor in `experimental`.